enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/CausalClusteringSettings.java

```diff
@@ -293,6 +293,11 @@
     public static final Setting<Duration> catch_up_client_inactivity_timeout =
             setting( "causal_clustering.catch_up_client_inactivity_timeout", DURATION, "20s" );
 
+    @Description( "Maximum retries per request during store copy. Regular store files and indexes are downloaded in separate requests during store copy." +
+            " This configures the maximum amount of retries per request before store copy will fail all together. " )
+    public static final Setting<Integer> store_copy_max_retries_per_request =
+            setting( "causal_clustering.store_copy_maximum_retries_per_request", INTEGER, "20" );
```

Comment:
Does this mean that the store copy will start over? If so, then I think this should be set much higher by default.

Reply:
It does. Its per request though.